### PR TITLE
Add input validation MD script

### PR DESCRIPTION
**Pull request by IAlibay**
A short click-based MD script that will validate an input PDB and cofactors SDF file.

What it doesn't do:
- Validate the ligand SDF
- Validate any ligand network file

The hope is that the ligands should all be fine and the network file will be generated in phase two.

### CODE REVIEW

**Comment by IAlibay** [comment]
The main todo left here is to put in a short pytest run of this script.

**Comment by IAlibay** [comment]
sorry @hannahbaumann, I re-organized the layout whillst I was at it (that way we can run tests _if_ we want). Hopefully it's not too many files - main one to look at is the utility script

File: industry_benchmarks/utils/input_validation.py [hunk 1]
```diff
@@ -0,0 +1,114 @@
...
+      A Path to an SDF file containing the system's cofactors.
+    """
+    # Create the solvent and protein components
+    solv = openfe.SolventComponent()
+    prot = openfe.ProteinComponent.from_pdb_file(str(pdb))
+
+    # Store there in a components dictionary
+    components_dict = {
+        'protein': prot,
+        'sovlent': solv,
```
Comment (hannahbaumann) [inline industry_benchmarks/utils/input_validation.py:87]:
```suggestion
        'solvent': solv,
```

File: industry_benchmarks/utils/tests/test_input_validation.py [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+                ['--pdb', protein, '--cofactors', cofactors]
+            )
+            assert result.exit_code == 0
+            assert "SIMULATION COMPLETE" in result.output
+
+    def test_bad_protein(self, bad_protein):
+        runner = click.testing.CliRunner()
+        with runner.isolated_filesystem():
+            result = runner.invoke(main, ['--pdb', bad_protein])
+            assert result.exit_code == 0
```
Comment (hannahbaumann) [inline industry_benchmarks/utils/tests/test_input_validation.py:54]: Is this protein supposed to run successfully or should this give an error?

Comment (IAlibay) [reply industry_benchmarks/utils/tests/test_input_validation.py:54]: ah that's a good point, it should fail! 👀

**Comment by IAlibay** [comment]
Should be ready for re-review @hannahbaumann :)

**Comment by IAlibay** [comment]
As mentioned on slack, I'm going to go ahead and merge this to update the repo layout, but please do still review if you can and I'll address things in a follow-up PR.

**Review by hannahbaumann — COMMENTED** [review]
lgtm!

**Pull request merged**